src/schema/v2/user.ts

```diff
@@ -184,6 +184,7 @@ export const UserType = new GraphQLObjectType<any, ResolverContext>({
       name: {
         description: "The given name of the user.",
         type: new GraphQLNonNull(GraphQLString),
+        resolve: ({ name }) => name || "Artsy User",
```

Comment:
Given the past headaches, I'd love to see a brief explanatory comment here.

Reply:
Nevermind I'll suggest the edit instead:

```suggestion
        resolve: ({ name }) => name || "Artsy User", // users may lack names, so fall back this non-null field
```